tests/sentry/api/serializers/test_project.py

```diff
@@ -140,7 +140,14 @@ def test_project_batch_has(self, mock_batch):
         assert "test-feature" in result["features"]
         assert "disabled-feature" not in result["features"]
 
-    def test_project_features(self):
+    @mock.patch("sentry.api.serializers.project.features")
+    def test_project_features(self, mock_features):
+        test_features = features.FeatureManager()
+        mock_features.all = test_features.all
+        mock_features.has = test_features.has
+        mock_features.batch_has = test_features.batch_has
+        mock_features.has_for_batch = test_features.has_for_batch
```

Comment:
I'm open to a more graceful way to do this. It feels ugly to have to catch every function that the code under test calls from the `features.*` namespace.